mappings/net/minecraft/util/dynamic/EntryLoader.mapping

```diff
@@ -2,20 +2,43 @@ CLASS net/minecraft/class_5506 net/minecraft/util/dynamic/EntryLoader
 	METHOD method_31154 resourceBacked (Lnet/minecraft/class_3300;)Lnet/minecraft/class_5506;
 		ARG 0 resourceManager
 	METHOD method_31155 load (Lcom/mojang/serialization/DynamicOps;Lnet/minecraft/class_5321;Lnet/minecraft/class_5321;Lcom/mojang/serialization/Decoder;)Ljava/util/Optional;
+		ARG 1 json
 		ARG 2 registryId
 		ARG 3 entryId
+		ARG 4 decoder
 	METHOD method_31156 getKnownEntryPaths (Lnet/minecraft/class_5321;)Ljava/util/Collection;
 		COMMENT @return A collection of file Identifiers of all known entries of the given registry.
 		COMMENT Note that these are file Identifiers for use in a resource manager, not the logical names of the entries.
 		ARG 1 key
 	CLASS 1
+		FIELD field_36186 JSON Ljava/lang/String;
 		METHOD method_31158 (Ljava/lang/String;)Z
 			ARG 0 name
+		METHOD method_39745 (Ljava/lang/String;Ljava/util/Set;Lnet/minecraft/class_5321;Lnet/minecraft/class_2960;)V
+			ARG 3 id
+		METHOD method_39746 createId (Lnet/minecraft/class_5321;Lnet/minecraft/class_5321;)Lnet/minecraft/class_2960;
+			ARG 0 rootKey
+			ARG 1 key
+		METHOD method_39747 getPath (Lnet/minecraft/class_5321;)Ljava/lang/String;
+			ARG 0 registryKey
 	CLASS class_5507 Impl
 		FIELD field_26741 values Ljava/util/Map;
+		FIELD field_36187 LOGGER Lorg/apache/logging/log4j/Logger;
 		METHOD method_31159 add (Lnet/minecraft/class_5455$class_5457;Lnet/minecraft/class_5321;Lcom/mojang/serialization/Encoder;ILjava/lang/Object;Lcom/mojang/serialization/Lifecycle;)V
 			ARG 1 registryManager
 			ARG 2 key
+			ARG 3 encoder
 			ARG 4 rawId
 			ARG 5 entry
 			ARG 6 lifecycle
+		METHOD method_39748 (Lnet/minecraft/class_5506$class_5507$class_6821;Ljava/lang/Object;)Lnet/minecraft/class_5506$class_6822;
+			ARG 1 value
+		METHOD method_39749 (Lnet/minecraft/class_5321;Lnet/minecraft/class_5321;)Ljava/util/stream/Stream;
+			ARG 1 registryKey
+		CLASS class_6821 Element
+	CLASS class_6822 IntIdentifiable
```

Comment:
This class doesn't look similar to `StringIdentifiable`. It looks more like a parse result holder.